### PR TITLE
Cheek Colors

### PR DESCRIPTION
**Pull request by thuyvyng**
got rid of background opacity and did it for cheeks instead, i think we ever need to make opacity more drastic or do diff cheek colors tho bc between my short medium and long i cant tell a diff -- i think this is fine until we decide how we want to proceed
Screen Shot 2023-01-24 at 2 25 21 PM

### CODE REVIEW

**Comment by thuyvyng** [comment]
i put a screenshot of all my avg loudness stuff, idk how much they differ for u -- what do u get

File: src/pages/userstats.js [hunk 1]
```diff
@@ -397,10 +373,11 @@ export default function UserStats() {
   }
 
   // background opacity ranges from 0.0 to 1.0, loudness ranges from about -65 to 0 decibels, so 1.0 opacity == loud
-  function computeBackgroundOpacity() {
+  function computeCheeksOpacity() {
     let loudnessArray = audioFeatures.map((item) => item.loudness);
     let averageLoudness = average(loudnessArray);
-    return (averageLoudness + 65) * (1 / 65);
+    console.log(averageLoudness);
```
Comment (thuyvyng) [inline src/pages/userstats.js:379]: left this here incase u wanna compare loudness values to decide if we still wanna do loudness

**Comment by thuyvyng** [comment]
Screen Shot 2023-02-02 at 2 48 36 PM
i like pink more

**Review by ruangroc — APPROVED** [review]
pink is definitely cuter

**Pull request merged**